darwin/dataset/download_manager.py

```diff
@@ -670,9 +674,44 @@ def _get_planned_image_paths(
             return [images_path / filename]
     else:
         for slot in annotation.slots:
+            if len(slot.source_files) > 1:
+                # Check that the item is either a DICOM series or a frame extracted from a video
+                is_dicom_series = all(
+                    source_file["file_name"].lower().endswith(".dcm")  # type: ignore
+                    for source_file in slot.source_files
+                )
+                is_extracted_frame = (
+                    len(slot.source_files) == 2
+                    and any(
+                        source_file["file_name"].lower().endswith(ext)  # type: ignore
+                        for ext in SUPPORTED_VIDEO_EXTENSIONS
+                        for source_file in slot.source_files
+                    )
+                    and any(
+                        source_file["file_name"].lower().endswith(ext)  # type: ignore
```

Comment:
could loop over SUPPORTED_IMAGE_EXTENSIONS and SUPPORTED_VIDEO_EXTENSIONS in the same any